docs/io/fits/appendix/faq.rst

```diff
@@ -573,11 +568,11 @@ of thousands of FITS files in succession (in either case the difference is
 not even an order of magnitude).
 
 Where data is concerned the situation is a little more complicated, and
-requires some understanding of how PyFITS is implemented versus CFITSIO and
+requires some understanding of how Astropy is implemented versus CFITSIO and
```

Comment:
When "Astropy" is used in such context, it is clearer to directly reference `astropy.io.fits`. Strictly speaking:
`PyFITS == astropy.io.fits` (not Astropy)